src/plugins/syscalls/syscalls.cpp

```diff
@@ -441,13 +510,45 @@ static event_response_t win_cb(drakvuf_t drakvuf, drakvuf_trap_info_t* info)
         }
     }
 
-    print_header(s->format, drakvuf, info);
-    if ( nargs )
+    // store informations we need to pass to win_ret_cb()
+    wrapper->args = (unsigned char*)g_malloc(sizeof(char)*size);
+    memcpy(wrapper->args, buf, sizeof(char)*size);
+    wrapper->wsc = wsc;
+    wrapper->target_cr3 = info->regs->cr3;
```

Comment:
We recently saw some issues trying to follow processes by cr3, it is actually better to do it by PID